Lib/test/test_super.py

```diff
@@ -342,6 +343,18 @@ def test_super_argtype(self):
             super(1, int)
 
     def test_shadowed_global(self):
+        source = textwrap.dedent(
+            """
+            class super:
+                msg = "truly super"
+
+            class C:
+                def method(self):
+                    return super().msg
+            """,
+        )
+        with import_helper.ready_to_import(name="shadowed_super", source=source):
+            import shadowed_super
         self.assertEqual(shadowed_super.C().method(), "truly super")
```

Comment:
Would it be possible to unload the module explicitly after its usage? Use unload() of import_helper.